frontend/src/metabase/common/hooks/use-entity-list-query/use-entity-list-query.unit.spec.tsx

```diff
@@ -165,6 +165,11 @@ describe("useEntityListQuery", () => {
     await waitFor(() => {
       expect(fetchMock.calls("path:/api/table")).toHaveLength(2);
     });
+
+    await waitFor(() => {
```

Comment:
this unit test seemed to be flaky so I added this waitFor so that I could focus on the connection errors

Reply:
are you sure it's flaky? I saw it failed on requests to localhost

Reply:
it failed a few times on me when I was working on this pr, but not always, I tried to run only that test a few times in a row and sometimes it passed and sometimes it failed.
Somebody edited this file since then though